package.json

```diff
@@ -43,7 +43,7 @@
       "^.+\\.(ts|tsx)$": "<rootDir>/preprocessor.js"
     },
     "testMatch": [
-      "**/test/*.(ts|js)"
+      "**/test/*.js"
```

Comment:
how about `**/test/*.ts`? In that way, we can run test without compiling ts to js explicitly.